src/main/java/org/prebid/server/bidder/rubicon/RubiconBidder.java

```diff
@@ -418,13 +423,25 @@ private boolean isMetricSupported(Metric metric) {
         return supportedVendors.contains(metric.getVendor()) && Objects.equals(metric.getType(), "viewability");
     }
 
-    private RubiconImpExt makeImpExt(Imp imp, ExtImpRubicon rubiconImpExt, Site site, App app) {
+    private RubiconImpExt makeImpExt(Imp imp,
+                                     ExtImpRubicon rubiconImpExt,
+                                     Site site,
+                                     App app,
+                                     ExtRequest extRequest) {
+        final ExtRequestPrebid extRequestPrebid = extRequest != null ? extRequest.getPrebid() : null;
+        final List<ExtRequestPrebidMultiBid> multibid = extRequestPrebid != null
+                ? extRequestPrebid.getMultibid() : null;
+        final ExtRequestPrebidMultiBid extRequestPrebidMultiBid =
+                CollectionUtils.emptyIfNull(multibid).stream().findFirst().orElse(null);
```

Comment:
It is too expensive to create stream for such kind of simple operation. Since we know the content, pls get it by idex if non-empty.